src/http/any-catchall/_fingerprint-paths.mjs

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs'
+import path from 'path'
+import url from 'url'
+const _local = process.env.ARC_ENV === 'testing'
```

Comment:
@ryanblock  @brianleroux is this the preferred way to identify local development?
Do we want to use ARC_LOCAL for this or did we get rid of that env var?

Reply:
For this case yeah, this is correct!